src/lib/libstrings.js

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: MIT
  */
 
-#include "arrayUtils.js"
+#include "../arrayUtils.js"
```

Comment:
This might affect user code, I worry? They would need to include one of our libraries from one of theirs, but still. We could avoid that risk by making `#include` add the extra directory layer?

Reply:
OK, I think I can make this work by using and include directory that includes `src` and `.`

Reply:
I update the include search mechanism to handle both cases.. I'm going to split that into its own PR I think.